tests/topotests/lib/ospf.py

```diff
@@ -22,6 +22,7 @@
 import ipaddr
 import ipaddress
 import sys
+import pdb
```

Comment:
leftover import from debugging?

Reply:
yes, but felt having it wont harm.

Reply:
Loading the `pdb` module registers hooks in the python interpreter for debugging, and can change behaviour of the remaining python code even if no functions from `pdb` are used.  Please remove it.

Reply:
Done